bigquery_etl/cli/query.py

```diff
@@ -1420,7 +1420,7 @@ def _initialize(query_file):
             except NotFound:
                 # continue with creating the table
                 pass
-        else:
+        elif len(materialized_views) == 0:
```

Comment:
Now that `materialized_view.sql` files are included in the file search, they'll be passed in directly to this function and we could just check the `query_file`:
```suggestion
        elif query_file.name != "materialized_view.sql":
```
And this function could also be refactored to not use the `materialized_views` list in general.